script/core.hbs

```diff
@@ -28,10 +30,32 @@
         {{/if}}
       {{/with}}
     };
+    {{#if global_config.useJWT}}
+      const jwtNotProvidedTimeout = setTimeout(() => {
+        console.warn(
+          'A JWT is required to initialize this Answers experience because "useJWT" is set to true.\n' +
```

Comment:
Should we change this to say something along the lines of "a JWT has not been received within 5s of page load"

Reply:
Ya I think that makes sense. I'll check with Rose on this because that was the error message originally requested in the spec

Reply:
ah I didn't realize the error message was from the spec, thought it was just a nice thing you came up with